aptos-move/e2e-move-tests/tests/aggregator.rs

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_types::transaction::SignedTransaction;
+use e2e_move_tests::{
+    aggregator::{
+        add, add_and_materialize, check, destroy, initialize, materialize, materialize_and_add,
+        materialize_and_sub, new, sub, sub_add, sub_and_materialize,
+    },
+    assert_abort, assert_success, MoveHarness,
+};
+use language_e2e_tests::account::Account;
+
+mod common;
+
+fn setup() -> (MoveHarness, Account) {
+    initialize(common::test_dir_path("aggregator.data/pack"))
+}
+
+#[test]
+fn test_aggregators_e2e() {
```

Comment:
Should we add a test to check the delete logic? Say I try to add to a destroyed aggregator?

Reply:
You cannot add to destroyed aggregator though, because the function consumes the value?

Reply:
Actually I can still check if it is in the table or not